src/middlewared/middlewared/plugins/network.py

```diff
@@ -476,6 +476,25 @@ def ipv4_in_use(self):
 
         return list_of_ipv4
 
+    @accepts()
+    def ipv6_in_use(self):
```

Comment:
This seem to duplicate quite a bit of code.

How about a general `ip_in_use`, then ipv6_in_use and ipv4_in_use could just call that, changing netif.AddressFamily.INET6 ?